content/docs/user-guide/testing/quality/_index.md

```diff
@@ -0,0 +1,216 @@
+                },
+                "q_graphics": {
+                    "Description": "Graphics quality settings.  0 : Low, 1 : Medium, 2 : High, 3 : VeryHigh",
+                    "Levels": [
+                        "Low",
+                        "Medium",
+                        "High",
+                        "VeryHigh"
+                    ],
+                    "Default": 3,
+                    "Settings": {
+                        "q_shadows": [ 0, 1, 2, 3 ]
+                    }
+                },
+                "q_shadows": {
+                    "Description": "Shadow quality settings.  0 : Low, 1 : Medium, 2 : High, 3 : VeryHigh",
+                    "Levels": [
+                        "Low",
+                        "Medium",
+                        "High",
+                        "VeryHigh"
+                    ],
+                    "Settings": {
+                        // shadows console variable settings
+                    }
+                }
+            }
+        }
+    }
+}
+```
+
+In the example above, when the user sets the quality level for the `q_general` quality group to `Low` or `0` the quality level for `q_graphics` and `q_shadows` will also be set to that level.
+
+
+## Quality levels 
+
+Quality levels can be defined in Settings Registry JSON files within each quality group JSON object.  
+
+Quality levels within each group JSON object should be defined in the following format.
+```json
+{
+    "O3DE": {
+        "Quality": {
+            "Groups": {
+                "q_general": {
+                    "Levels": [ // quality level names should be a single word
+                        "Low",
+                        "Medium",
+                        "High",
+                        "VeryHigh"
+                        ...
+                    ]
+                }
+            }
+        }
+    }
+}
+```
+
+Each group can have it's own quality levels defined with custom names and number.  Providing quality level names other than `Low`, `High`, etc. can be useful when defining levels for settings that aren't necessarily better or worse, for example you may have settings for power consumption levels like `HighPerformance`, `Balanced`, `EnergyEfficient`. 
+
+## Customizing quality levels for your project
+
+To override and customize the quality settings in the engine and the gems your project uses, define your own `.setreg` file in your projects `Registry` folder that contains the appropriate JSON content.  For example, if you wanted to override the default quality level to use in your game to be `VeryHigh` and provide a specific shadow console variable for a specific quality level you could add the following `.setreg` file to your project.
+
+### Project/Registry/quality.setreg
+
+```json
+{
+    "O3DE": {
+        "Quality": {
+            "Groups": {
+                "q_general": {
+                    "Default": 3 // change the default level to use to be VeryHigh instead of High
```

Comment:
nit:
`// change the default level to use to be VeryHigh instead of High`
to
`// change the default level to VeryHigh instead of High`